src/main/kotlin/com/hibob/academy/dao/tables.kt

```diff
@@ -0,0 +1,26 @@
+package com.hibob.academy.dao
+
+import com.hibob.academy.utils.JooqTable
+
+class PetTable(tableName: String = "pet"): JooqTable(tableName) {
+
+    val name = createVarcharField("name")
+    val type = createVarcharField("type")
+    val companyId = createBigIntField("company_id")
+    val dateOfArrival = createDateField("date_of_arrival")
+
+    companion object {
+        val instance = PetTable()
+    }
+}
+
+class OwnerTable(tableName: String = "owner"): JooqTable(tableName) {
+
```

Comment:
What about id field?
Do we want to query by id as well? If so then we need to add it as well here.